go/ql/lib/semmle/go/security/FileSystemAccess.qll

```diff
@@ -0,0 +1,225 @@
+      ) and
+      this = m.getACall()
+    )
+  }
+
+  override DataFlow::Node getAPathArgument() { result = this.getArgument(pathArg) }
+}
+
+/**
+ * Provide File system access sinks of [beego](https://github.com/beego/beego) web framework
+ */
+class EchoFileSystemAccess extends FileSystemAccess::Range, DataFlow::CallNode {
+  int pathArg;
+
+  EchoFileSystemAccess() {
+    exists(Method m |
+      m.hasQualifiedName(package("github.com/labstack/echo", ""), "Context", ["Attachment", "File"]) and
+      this = m.getACall() and
+      pathArg = 0
+    )
+  }
+
+  override DataFlow::Node getAPathArgument() { result = this.getArgument(pathArg) }
+}
+
+/**
+ * The File system access sinks of [gin](https://github.com/gin-gonic/gin) web framework
+ */
+class GinFileSystemAccess extends FileSystemAccess::Range, DataFlow::CallNode {
+  int pathArg;
+
+  GinFileSystemAccess() {
+    exists(Method m |
+      (
+        m.hasQualifiedName(package("github.com/gin-gonic/gin", ""), "Context",
+          ["File", "FileAttachment"]) and
+        pathArg = 0
+        or
+        m.hasQualifiedName(package("github.com/gin-gonic/gin", ""), "Context", "SaveUploadedFile") and
+        pathArg = 1
+      ) and
+      this = m.getACall()
+    )
+  }
+
+  override DataFlow::Node getAPathArgument() { result = this.getArgument(pathArg) }
+}
+
+/**
+ * The File system access sinks of [iris](https://github.com/kataras/iris) web framework
+ */
+class IrisFileSystemAccess extends FileSystemAccess::Range, DataFlow::CallNode {
+  int pathArg;
+
+  IrisFileSystemAccess() {
+    exists(Method m |
+      (
+        m.hasQualifiedName(package("github.com/kataras/iris", "context"), "Context",
+          ["SendFile", "ServeFile", "SendFileWithRate", "ServeFileWithRate", "UploadFormFiles"]) and
+        pathArg = 0
+        or
+        m.hasQualifiedName(package("github.com/kataras/iris", "context"), "Context", "SaveFormFile") and
+        pathArg = 1
+      ) and
+      this = m.getACall()
+    )
+  }
+
+  override DataFlow::Node getAPathArgument() { result = this.getArgument(pathArg) }
+}
+
+/**
+ * The File system access sinks of [fiber](https://github.com/gofiber/fiber) web framework
+ */
+class FiberSystemAccess extends FileSystemAccess::Range, DataFlow::CallNode {
+  int pathArg;
+
+  FiberSystemAccess() {
+    exists(Method m |
+      (
+        m.hasQualifiedName(package("github.com/gofiber/fiber", ""), "Ctx", "SendFile") and
+        pathArg = 0
+        or
+        m.hasQualifiedName(package("github.com/gofiber/fiber", ""), "Ctx", "SaveFile") and
+        pathArg = 1
+      ) and
+      this = m.getACall()
+    )
+  }
+
+  override DataFlow::Node getAPathArgument() { result = this.getArgument(pathArg) }
+}
+
+/**
+ * Provide File system access sinks of [afero](https://github.com/spf13/afero) framework
+ * The Types that are not vulnerable: `afero.BasePathFs` and `afero.IOFS`
+ */
+module Afero {
+  string aferoPackage() { result = "github.com/spf13/afero" }
```

Comment:
```suggestion
  string aferoPackage() { result = package("github.com/spf13/afero", "") }
```
Then you can replace all the usages of `package(aferoPackage(), "")` with `aferoPackage()`.